Purchases/SubscriberAttributes/RCSubscriberAttributesManager.m

```diff
@@ -0,0 +1,137 @@
+//
+// Created by RevenueCat on 2/7/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+#import "RCSubscriberAttributesManager.h"
+#import "RCSubscriberAttribute.h"
+#import "RCSpecialSubscriberAttributes.h"
+#import "RCBackend.h"
+#import "RCDeviceCache.h"
+#import "NSError+RCExtensions.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+
+@interface RCSubscriberAttributesManager ()
+
+@property (nonatomic) RCDeviceCache *deviceCache;
+@property (nonatomic) RCBackend *backend;
+
+@end
+
+
+@implementation RCSubscriberAttributesManager
+
+#pragma MARK - Public methods
+
+- (instancetype)initWithBackend:(nullable RCBackend *)backend
+                    deviceCache:(nullable RCDeviceCache *)deviceCache {
+    if (self = [super init]) {
+        self.backend = backend;
```

Comment:
question here: I'd usually assert that `backend != nil` and `deviceCache != nil`, so that a bug doesn't mean that the class all of a sudden no-ops. 
However, I wouldn't want a bug on our side to potentially cause clients to crash... But then again it'd crash right away, so the assert here might actually guarantee that we don't create the bug in the first place. 

Thoughts?

Reply:
sounds good, adding

Reply:
I think this hasn't been added yet

Reply:
forgot to push 🤦‍♂  :hides in shame: